### PR TITLE
updated and fixed CI configs and versions

### PR DESCRIPTION
**Pull request by nzambello**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
-python:
-  - 2.7
+python: 2.7.13
+sudo: false
+cache:
+  pip: true
+  directories:
+    - eggs
+env:
+  - PLONE_VERSION=5.1.x
```
Comment (pnicolli) [inline .travis.yml]: What about also checking plone 5.0.x ? We are actually using this with plone 5.0.8.

File: buildout.cfg [hunk 2]
```diff
@@ -1,11 +1,17 @@
 [buildout]
-extends = http://dist.plone.org/release/4.3.6/versions.cfg
 extensions = mr.developer
+
+extends =
+    plone-5.1.x.cfg
+    versions.cfg
```
Comment (pnicolli) [inline buildout.cfg]: If I understood correctly how it works, the plone versione here gets replaced by sed during CI, so I guess this should be `plone-x.x.x.cfg`. But then how do we buildout locally? What am I missing here?
Moreover, I see that `versions.cfg` is also extended inside `plone-5.1.x.cfg`, I guess it's not necessary to extend it twice.

File: .travis.yml (hunk 1 above)
Comment (nzambello) [reply .travis.yml]: So you're suggesting to test both, aren't you?

File: buildout.cfg (hunk 2 above)
Comment (nzambello) [reply buildout.cfg]: Now this is fixed and there are no duplicates of extends

File: .travis.yml (hunk 1 above)
Comment (nzambello) [reply .travis.yml]: Ok, added both in the env

**Comment by cekk** [comment]
thanks, but there are some things that we don't need (like base.cfg file).
btw there was already a branch for tests, and now a pr: #16

**Pull request closed without merging**